src/css/main.scss

```diff
@@ -1382,3 +1382,43 @@ div.search-input:focus-within {
     height: 20px;
   }
 }
+
+// New feature bar styling
+.new-feature-bar {
+  height: 50px;
+  background-color: var(--blue-primary);
+  color: #FFFFFF;
+  width: 100%;
+  padding: 12px 32px;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 17px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  svg {
+    width: 18px;
+    height: 18px;
+  }
+
+  .version-box {
+    background-color: var(--blue-100);
+    border-radius: 40px;
+    padding: 4px 8px;
+    color: var(--blue-primary);
+    display: flex;
+    align-items: center;
+    gap: 4px;
+
+    svg {
+      width: 18px;
+      height: 18px;
+    }
```

Comment:
I am not sure if this `svg` styles is not inherited from the `svg` above. If not you can also add a className to it to remove this additional styles.

Reply:
yeah, I just needed it in the `.version-box` at the beginning and then it turned out to be needed also "globally", so I just forgot to remove this part. Thanks.